### DISCUSSION TITLE
How to handle "2.7.x stable" and "develop" translation projects in Transifex in a long run

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [keepassxreboot/keepassxc#9711] [Q&A] [answered]
Hey there!

I'm translator, obviously. What's not clear to me is how those two projects co-exist? My guess is that `develop` is the long-living one and from time to time new `n.n.x stable` is being spun off of it. Or is it the other way around?

From Transifex notifications I can tell that both projects do receive updates.

I'm asking because in a long term I'm bringing the project translation into shape and that'll be a continued effort that'll probably span across several releases (yes, I'm that slow). My desire is to not maintain two parallel projects that differ on several (or several thousand) strings but to focus my efforts on a single one that'll be base for the next `n.n.x stable` (or `develop`) at some point in the future. Does that makes sense?

**Comment by phoerious** [comment, marked as answer]
The stable version should receive only minor updates. The develop contains the next big feature release and may diverge significantly from the stable branch in time.
